how-it-works/1_protocol/00_protocol_overview.subpage.md

```diff
@@ -14,6 +14,21 @@ Canister code execution updates the canister state.
 In order to keep the state on the subnet nodes on which a canister is hosted in sync, it must be ensured that every node executes the same messages in the same order, i.e., fully deterministically.
 This is the core of the blockchain-based replicated state machine functionality realizing the IC.
 
+Each node on the Internet Computer runs a replica process. The replica process is structured in a layered architecture consisting of the following 4 layers.   
+1. Peer-to-peer
+2. Consensus
+3. Message routing
+4. Execution
+
+<figure>
+<img src="/img/how-it-works/core_protocol_layers.png" alt="4-layer architecture of the Internet Computer" title="4-Layer Core IC Protocol" align="center" style="width:600px">
```

Comment:
something is still wonky with the alignment. Can you try to fix it, please?

Reply:
Still not quite working: 
<img width="1190" alt="image" src="https://user-images.githubusercontent.com/14757975/201979651-81baeb78-f645-458a-bf9b-f2e600aac196.png">

Reply:
Fixed it